src/main/java/entity/Task.java

```diff
@@ -0,0 +1,33 @@
+package entity;
+
+/**
+ * Abstract class which would be inherited by the various subclasses Deadline, ToDo and Event
+ */
+public abstract class Task {
+    protected String title;
```

Comment:
Why is the title and marked here protected? whereas deadline and event class fields are private? Perhaps could standardise? Also perhaps it might be more clear to name boolean with is___, in this case, isMarked? :)